File: src/main/java/org/acme/mongodb/panache/entities/Person.java
package org.acme.mongodb.panache.entities;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.quarkus.mongodb.panache.PanacheMongoEntity;

import javax.transaction.Status;
import java.time.LocalDate;

public class Person extends PanacheMongoEntity {
    @JsonProperty
    public String name;
    public String status;
    public int account;
}

File: src/main/java/org/acme/mongodb/panache/entities/Person.kt
package org.acme.mongodb.panache.entities

import com.fasterxml.jackson.annotation.JsonProperty
import io.quarkus.mongodb.panache.PanacheMongoEntity

class Person : PanacheMongoEntity() {
    @JsonProperty
    var name: String? = null
    var status: String? = null
    var account = 0
}

fun test(valor1: String, valor2:String= "test"){

}

fun another(){
    test(valor1 = "hola")

}
